README.md

```diff
@@ -29,6 +30,7 @@ region = "us-west-1"
 host = "Server1"
 whitelist_pass_through = False
 debug = False
+proxy_server_name = "http://myproxyserver.com:8080"
```

Comment:
I will split proxy server name and server port. Currently they both are together.